### PR TITLE
numaresources: add -tests image for CI

### PR DESCRIPTION
**Pull request by ffromani**
build the -tests image, to be used on CIs.

Signed-off-by: Francesco Romani <fromani@redhat.com>

### CODE REVIEW

**Comment by fromanirh** [comment]
/hold
depends on https://github.com/openshift-kni/numaresources-operator/pull/165 (and needs changes after 165 merges)

**Comment by fromanirh** [comment]
/hold cancel
updated on top of https://github.com/openshift-kni/numaresources-operator/pull/165

File: ci-operator/config/openshift-kni/numaresources-operator/openshift-kni-numaresources-operator-main.yaml [hunk 1]
```diff
@@ -26,6 +26,14 @@ images:
       - destination_dir: .
         source_path: /go/src/github.com/openshift-kni/numaresources-operator/bin/exporter
   to: resource-topology-exporter
+- dockerfile_path: Dockerfile.openshift.tests
+  from: os-min8
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/src/github.com/openshift-kni/numaresources-operator/bin/.
```
Comment (yanirq) [inline ci-operator/config/openshift-kni/numaresources-operator/openshift-kni-numaresources-operator-main.yaml:35]: is the` /.` at the end the right format ?

Comment (Tal-or) [reply ci-operator/config/openshift-kni/numaresources-operator/openshift-kni-numaresources-operator-main.yaml:35]: If the CI passed, I think it is

**Pull request merged**